src/coreclr/nativeaot/System.Private.CoreLib/src/System/AppContext.NativeAot.cs

```diff
@@ -36,5 +37,18 @@ internal static void OnUnhandledException(object e)
         {
             UnhandledException?.Invoke(/* AppDomain */ null, new UnhandledExceptionEventArgs(e, true));
         }
+
+        private static unsafe string GetRuntimeModulePath()
+        {
+            delegate*<string> ip = &GetRuntimeModulePath;
```

Comment:
I would strip this function pointer with a cast to `IntPtr` or `void*`. Yes, it is a function pointer, but it isn't being called and passing an managed function pointer instead of an unmanaged function pointer to what looks like a QCall sends one down a rabbit hole. A comment might be sufficient here, but it looks very suspicious at first glance.